skyflow/errors/_skyflow_errors.py

```diff
@@ -43,11 +44,18 @@ class SkyflowErrorMessages(Enum):
     INVALID_RECORDS_TYPE = "Records key has value of type %s, expected list"
     INVALID_FIELDS_TYPE = "Fields key has value of type %s, expected dict"
     INVALID_TABLE_TYPE = "Table key has value of type %s, expected string"
+    INVALID_TABLE_TYPE_DELETE = "Table of type string is required in records array"
     INVALID_IDS_TYPE = "Ids key has value of type %s, expected list"
     INVALID_ID_TYPE = "Id key has value of type %s, expected string"
+    INVALID_ID_TYPE_DELETE = "Id of type string is required at index 0 in records array"
```

Comment:
It shouldn't be hard coded as `index 0`. For scenarios where `id` is present 0th record, but not in nth record, Error message will be inconsistent  It should be `index %s` instead where index is populated dynamically.

Reply:
okay